pkgs/tools/admin/auth0-cli/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "auth0-cli";
+  version = "0.11.10";
+
+  src = fetchFromGitHub {
+    owner = "auth0";
+    repo = "auth0-cli";
+    rev = "v${version}";
+    hash = "sha256-1/T2hpSNamorWFuaSBoLsGpe9I06HGew9S3yJsDLmLQ=";
+  };
+
+  vendorHash = "sha256-d9ZwK/LAZGgeagGsg3bGYnVykfQcCLUex0pe/PUCtkA=";
+
+  meta = with lib; {
+    description = "Supercharge your developer workflow";
```

```suggestion
    description = "Build, test, troubleshoot and manage your integration with Auth0 directly from your terminal";
```

That description is very useless but that's auth0's fault. :upside_down_face: 
This seems a bit more useful & was in the README